orttraining/orttraining/core/framework/pipeline.h

```diff
@@ -125,9 +125,9 @@ class PipelineSlot {
   // Print this structure following a fixed-length format.
   // It assumes there are at most 2 actions per slot.
   friend std::ostream& operator<<(std::ostream& stream, const PipelineSlot& slot);
-  void SetWaitedEvent(const std::vector<int> event);
+  void SetWaitedEvent(const std::vector<int>& event);
```

Comment:
It should either be "const std::vector<int>& event" or "std::vector<int> event".  A const object shouldn't be moved.